### PR TITLE
RSLC-11- Admin users

### PR DESCRIPTION
**Pull request by RCarmona53**
#### Board:
* [RSLC-11](https://trello.com/c/TPpMM5G0/7-rslc-11-as-an-admin-i-should-be-able-to-see-all-users-in-the-system)
---
#### Description:
As an admin I should be able to see all users in the system
image

### CODE REVIEW

File: spec/admin/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  describe 'Show all users' do
+    it 'displays all users for the admin user' do
+      user1 = FactoryBot.create(:user, email: 'user1@example.com')
+      user2 = FactoryBot.create(:user, email: 'user2@example.com')
+
+      get '/admin/users'
+
+      expect(response).to have_http_status(200)
```
Comment (rodrieiz) [inline spec/admin/user_spec.rb]: we can use `be_ successful` here

**Review by rodrieiz — APPROVED** [review]

**Review by santiagodiaz — APPROVED** [review]

**Comment by rodrieiz** [comment]
If I'm not wrong the base branch in this pr is should be `master` because this feature not depends on `AdminUsers` card or pr.  wdyt?

I would like to give some general comments: 
As an advice if is possible to do a card independently of other branch do it, because you'll avoid to have to make merge or rebase many times. Maybe you also can merged quickly because don't have to wait for previous prs. 
And another advice try to reduce the number of commits, for example I saw in previous pr 'fix comment 1 ', 'fix comment 2' and very large list of commit for the same functionality. You can use  `git commit --amend` to edit the last commit 
[Commits RS Guidelines](https://github.com/rootstrap/tech-guides/blob/master/git/commits.md)

**Comment by rodrieiz** [comment]
@santiagodiaz I'm not sure what's going on but it's like a lot of files/commits are being spread from one pr to another with things that have already been resolved and merged. Maybe it's better to close and open the prs again.

File: spec/admin/user_spec.rb (hunk 1 above)
Comment (RCarmona53) [reply spec/admin/user_spec.rb]: Done!

**Comment by santiagodiaz** [comment]
> @santiagodiaz I'm not sure what's going on but it's like a lot of files/commits are being spread from one pr to another with things that have already been resolved and merged. Maybe it's better to close and open the prs again.

thanks @rodrieiz, you're right, we could solve this with Rodri by doing a rebase from master with the latest changes, so this PR contains just 2 file changes

**Pull request merged**